base/debloat.sh

```diff
@@ -39,7 +39,7 @@ debloat_paths=(
     "/nix"
 )
 
-if [[ ! "$PROFILES" == *"devtools"* ]]; then
```

Comment:
Without this i was getting: 
```
‣  Running finalize script /home/debian/mnt/base/debloat.sh…
/work/finalize: line 42: PROFILES: unbound variable
‣ "/work/finalize" returned non-zero exit code 1.
```
I'm not sure if there is any danger in accepting an unbound value here.